librespot-android-sink/src/main/java/xyz/gianlu/librespot/android/sink/AndroidSinkOutput.java

```diff
@@ -19,24 +19,51 @@
     private AudioTrack track;
     private float lastVolume = -1;
 
-    public AndroidSinkOutput() {
-    }
-
     @Override
     public boolean start(@NotNull OutputAudioFormat format) throws SinkException {
```

Comment:
Can I rely on the incoming OutputAudioFormat to have sane values or do I need to sanity check them?

Reply:
It's all checked when it comes in.